### DISCUSSION TITLE
Regarding GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by sharma-jayprakash** [community/community#138425] [GitHub Education]
### Hi everyone,

I verified for GitHub student developer pack but still I dont get it how much time it will gonna take and what benefits or things  I am going to get from the GitHub student developer pack.

**Comment by andrefqms** [comment]
It's around within 3 days of the approval status. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status[ here ](https://education.github.com/discount_requests/pack_application).

For any problem raise a ticket with [Education support](https://support.github.com/contact/education)

**Comment by davevad93** [comment]
Hi @sharma-jayprakash ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sharma-jayprakash  was approved and your benefits are now available.`

For more infos, check the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

As for the benefits you'll get with the `Student Developer Pack`, you can check the full list [here](https://education.github.com/pack).

**Reply by sharma-jayprakash** [reply]
But currently it's showing  pro![IMG-20240911-WA0007.jpg](https://github.com/user-attachments/assets/5c2841f0-9111-4a3f-8876-191135d6be9e)

**Reply by davevad93** [reply]
Hey @sharma-jayprakash, right now your status is green and it says your benefits will be available soon. As I said earlier the benefits will be available when the status label will turn purple, the waiting time is variable and considering you have been approved yesterday, you'll have to wait a few days minimum.

**Reply by ElisavetAmpatzidou** [reply]
Hi, my label is purple but I dont have access to pro features and my account status still be "Github Free".
![image](https://github.com/user-attachments/assets/272e3b03-78ae-4379-a1d2-74b2ed5c25c7)
![image](https://github.com/user-attachments/assets/6d9ca5ee-4611-47f6-a4b0-6bd4bd92baba)

Note: I have renew on 12th January my account because it was expired. Any ideas? 
Thank you in advance!

**Reply by davevad93** [reply]
Hi @ElisavetAmpatzidou , 

Check my response in the other post, also note that generally the student benefits lasts for 2 years, so technically you'll have to renew on Jan 12  2026.
